tools/pipeline_perf_test/orchestrator/lib/impl/strategies/deployment/docker.py

```diff
@@ -0,0 +1,189 @@
+from typing import Dict, List, Literal, Optional, Union
```

Comment:
Including this mostly for illustration, changes above were largely in support of making sure the hooks / strategies could access everything needed.